content/en/docs/tasks/administer-cluster/nodelocaldns.md

```diff
@@ -47,18 +48,43 @@ This is the path followed by DNS Queries after NodeLocal DNSCache is enabled:
 {{< figure src="/images/docs/nodelocaldns.jpg" alt="NodeLocal DNSCache flow" title="Nodelocal DNSCache flow" caption="This image shows how NodeLocal DNSCache handles DNS queries." >}}
 
 ## Configuration
-
-This feature can be enabled using the command:
-
-`KUBE_ENABLE_NODELOCAL_DNS=true kubetest --up`
-
-This works for e2e clusters created on GCE. On all other environments, the following steps will setup NodeLocal DNSCache:
-
-* A yaml similar to [this](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml) can be applied using `kubectl create -f` command.
-* No need to modify the --cluster-dns flag since NodeLocal DNSCache listens on both the kube-dns service IP as well as a link-local IP (169.254.20.10 by default)
+{{< note >}} The local listen IP address for NodeLocal DNSCache can be any IP in the 169.254.20.0/16 space or any other IP address that can be guaranteed to not collide with any existing IP. We have used 169.254.20.10 as an example in this document.
```

Comment:
nit:
```suggestion
{{< note >}} The local listen IP address for NodeLocal DNSCache can be any IP in the 169.254.20.0/16 space or any other IP address that can be guaranteed to not collide with any existing IP. This document uses 169.254.20.10 as an example.
```

I'd prefer to have that wording. Also (I've mentioned this a few times) I would prefer to use a [documentation IPv4 address](https://tools.ietf.org/html/rfc5737) rather than an address inside 169.254.20.0/16.

It would be OK to tell readers under what circumstances they might pick an address inside 169.254.20.0/16 as the actual address to replace the example address with.